dbfit-java/informix/src/main/java/dbfit/environment/InformixBigDecimalTransformer.java

```diff
@@ -0,0 +1,20 @@
+package dbfit.environment;
+
+import java.sql.SQLException;
+
+import dbfit.util.TypeTransformer;
+
+public class InformixBigDecimalTransformer implements TypeTransformer {
+
+    @Override
+    public Object transform(Object o) throws SQLException {
+        if (o == null) {
+            return null;
+        }
+        if (!(o instanceof dbfit.util.NormalisedBigDecimal)) {
+            throw new UnsupportedOperationException("InformixBigDecimalSpecifier cannot work with " + o.getClass());
```

Comment:
Java checks exception type during compile time (it won't compile if it's not OK). `RuntimeException` and derived ones are special case - they can be thrown by any method (no need to specify it in `throws` list).
`SQLException` I believe is intended for database-related errors.